tests/hazmat/primitives/utils.py

```diff
@@ -353,7 +353,7 @@ def hkdf_extract_test(backend, algorithm, params):
         backend=backend,
     )
 
-    prk = hkdf._extract(binascii.unhexlify(params["ikm"]))
+    prk = hkdf._extract(binascii.unhexlify(params["ikm"]))  # type: ignore[attr-defined]
```

Comment:
Uhh, is there a reason we have this test? I'd rather stop exposing this random (trivial) implementation detail

Reply:
We probably only have it because it was possible to test intermediate values so we did. I'll remove it in a separate PR and then rebase this one.